app/experimenter/nimbus-ui/src/components/FormAudience/index.tsx

```diff
@@ -0,0 +1,196 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import Select from "react-select";
+import Form from "react-bootstrap/Form";
+import InputGroup from "react-bootstrap/InputGroup";
+import Col from "react-bootstrap/Col";
+import LinkExternal from "../LinkExternal";
+
+import { getExperiment_experimentBySlug } from "../../types/getExperiment";
+import {
+  getConfig_nimbusConfig,
+  getConfig_nimbusConfig_channels,
+} from "../../types/getConfig";
+
+// TODO: find this doco URL
```

Comment:
I think we have a couple of these "find link" TODOs now, one in Audience and one in Branches. We should probably have a follow up ticket made to find/fix these.

Reply:
Filed as _"Find+fix all the TODO "find link" documentation URLs #4101"_.